python/nav/ipdevpoll/daemon.py

```diff
@@ -262,13 +282,16 @@ def make_option_parser(self):
         opt("-p", "--list-plugins", action="store_true",
             help="load and print a list of configured plugins")
         opt("-J", action="store", dest="onlyjob", choices=self._joblist(),
-            metavar="JOBNAME", help="run only JOBNAME in this process")
+            metavar="JOBNAME", help="run only JOBNAME jobs in this process")
         opt("-n", "--netbox", action=NetboxAction, metavar="NETBOX",
             help="Run JOBNAME once for NETBOX. Also implies -f and -s options.")
         opt("-m", "--multiprocess", type=int, dest="multiprocess",
             nargs='?', const=cpu_count(), metavar='WORKERS',
             help="Run ipdevpoll in a multiprocess setup. If WORKERS is not set "
             "it will default to number of cpus in the system")
+        opt("-T", "--max-jobs-per-worker", type=int, dest="max_jobs",
```

Comment:
I've only one comment: `-T` doesn't seem intuitive to me as a shorthand for `--max-jobs-per-worker`. A capital `-M` would probably do.